ivy/functional/frontends/jax/__init__.py

```diff
@@ -1,3 +1,5 @@
 # flake8: noqa
 from .lax import operators
 from ivy.functional.frontends.jax.lax.operators import *
+from .numpy import functions
```

Comment:
I should have made this clear, but we are actually only concerned with adding the `jax.lax` frontend for the time being, and not the `jax.numpy` namespace. If you want to add a similar frontend function for `jax` as well, then you could add [jax.lax.full](https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.full.html#jax.lax.full)